### ISSUE TITLE
RN The official scaffolding project introduced bundlejs and modified it again, but the hot update could not be triggered

### ISSUE THREAD

**Issue by nianxiongdi** [facebook/react-native#44890]
### Description

1. Introduced in app.tsx. official scaffolding、official scaffolding、official scaffolding、official scaffolding
```js
const infos = require('./infos')



const App = () => {

  <View>

  <<infos />
</view>
}
     
export default App;

```

2. Modify file A and cannot trigger hot.

3. I would like to ask how to trigger my bundlejs file through metro and trigger hot update again.

4. Thank you very much.

### React Native Version

0.73.0

### Affected Platforms

Runtime - iOS
 

### Stacktrace or Logs

```text
none
```

### Reproducer

none

### Screenshots and Videos

none

**Comment by nianxiongdi** [comment]
> ⚠️	Newer Version of React Native is Available!
> ℹ️	You are on a supported minor version, but it looks like there's a newer patch available - 0.73.8. Please [upgrade](https://reactnative.dev/docs/upgrading) to the highest patch for your minor or latest and verify if the issue persists (alternatively, create a new project and repro the issue in it). If it does not repro, please let us know so we can close out this issue. This helps us ensure we are looking at issues that still exist in the most recent releases.

upgrading to the latest version still has problems.
```js
    "react": "18.2.0",
    "react-native": "0.74.1",
```

**Comment by nianxiongdi** [comment]
@cortinico  [Already asked, no one replied](https://stackoverflow.com/questions/78478765/rn-the-official-scaffolding-project-introduced-bundlejs-and-modified-it-again-b)
